recipe/meta.yaml

```diff
@@ -32,60 +32,63 @@ package:
   version: {{ dal_version }}
 
 source:
-  - url: https://anaconda.org/intel/mkl/{{ mkl_version }}/download/{{ target_platform }}/mkl-{{ mkl_version }}-intel_{{ mkl_buildnum }}.tar.bz2
+  - url: https://software.repos.intel.com/intel-conda/{{ target_platform }}/mkl-{{ mkl_version }}-intel_{{ mkl_buildnum }}.tar.bz2
```

Comment:
should we move channel to separate variable like it is done at https://github.com/conda-forge/intel-compiler-repack-feedstock ?

Reply:
agree - but would first integrate this one if ci would pass, and this can be fixed later on